core-kotlin-modules/core-kotlin-advanced-3/pom.xml

```
@@ -31,6 +31,22 @@
             <artifactId>classgraph</artifactId>
             <version>4.8.112</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.jetbrains.kotlinx</groupId>
+            <artifactId>kotlinx-coroutines-test</artifactId>
+            <version>1.7.3</version>
+            <type>pom</type>
```

Why is this a `pom` type? Wouldn't this just be a normal dependency? If anything, wouldn't this be a `<scope>test</scope>` dependency?

```suggestion
            <scope>test</scope>
```